### PR TITLE
Add .well-known/changepassword handler

### PR DESCRIPTION
**Pull request by rullzer**

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Looks good but where's that actually used?

File: apps/settings/composer/autoload.php [hunk 1]
```diff
@@ -3,8 +3,21 @@
 // autoload.php @generated by Composer
 
 if (PHP_VERSION_ID < 50600) {
-    echo 'Composer 2.3.0 dropped support for autoloading on PHP <5.6 and you are running '.PHP_VERSION.', please upgrade PHP or use Composer 2.2 LTS via "composer self-update --2.2". Aborting.'.PHP_EOL;
-    exit(1);
+    if (!headers_sent()) {
```
Comment (nickvergessen) [inline apps/settings/composer/autoload.php:6]: Seems unrelated?

**Review by nickvergessen — APPROVED** [review]
https://w3c.github.io/webappsec-change-password-url/

**Review by ChristophWurst — APPROVED** [review]

File: apps/settings/composer/autoload.php (hunk 1 above)
Comment (rullzer) [reply apps/settings/composer/autoload.php:6]: just my composer doing composer things. But I had to update the autoloader.

**Review by CarlSchwan — APPROVED** [review]
This is cool :)

**Comment by rullzer** [comment]
Somebody with merge permissions lets get this in. The tests are flaky so I can't.
